client/my-sites/marketplace/components/reviews-list/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
+import Rating from 'calypso/components/rating';
+import {
+	ProductProps,
+	useMarketplaceReviewsQuery,
+	MarketplaceReviewResponse,
+} from 'calypso/data/marketplace/use-marketplace-reviews';
+import './style.scss';
+import { sanitizeSectionContent } from 'calypso/lib/plugins/sanitize-section-content';
+
+export const MarketplaceReviewsList = ( props: ProductProps ) => {
+	const translate = useTranslate();
+	const { data: reviews } = useMarketplaceReviewsQuery( props );
+
+	if ( ! isEnabled( 'marketplace-reviews-show' ) ) {
```

Comment:
nitpick: this can be on the top

Reply:
As this is a condition, it cannot be added before calling the hooks (`useTranslate` and `useMarketplaceReviewsQuery`)  as they will be conditionally called and that breaks the hook's rules 😄